schema.d.ts

```diff
@@ -592,6 +592,10 @@ export interface Parameter extends PropertyLike {
    * Whether the parameter is optional. Undefined implies non-optional.
    */
   optional?: boolean;
+  /**
+   * Whether the parameter is a spread. Undefined implies single parameter.
+   */
+  spread?: boolean;
```

Comment:
I think this should be called "rest": https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters

"rest" is for the parameter declaration. "spread" is the for argument at the callsite. Spread arguments can be used for non-rest parameters, and rest parameters can be called with non-spread arguments.